### PR TITLE
Fix all headers

### PR DESCRIPTION
**Pull request by Lainow**

### CODE REVIEW

**Review by cedric-anne — COMMENTED** [review]
We usually use a GPLv3 for our new plugins. @orthagh Should we use MIT or GPLv3 for this plugin?

Example of headers template file used by a GPLv3 plugin: https://github.com/pluginsGLPI/centreon/blob/main/tools/HEADER

If you change the license, you will have to use this `LICENSE` file: https://github.com/pluginsGLPI/centreon/blob/main/LICENSE
https://github.com/pluginsGLPI/centreon/blob/main/LICENSE

**Comment by orthagh** [comment]
GPLv3

**Review by orthagh — APPROVED** [review]

File: tools/HEADER [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
+
+You should have received a copy of the GNU General Public License
+along with Deploy. If not, see <http://www.gnu.org/licenses/>.
 -------------------------------------------------------------------------
-@copyright Copyright (C) 2022 by the deploy plugin team.
-@license   MIT https://opensource.org/licenses/mit-license.php
-@link      https://github.com/pluginsGLPI/deploy
+@copyright Copyright (C) 2022-2023 by Deploy plugin team.
+@license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+@link      https://github.com/pluginsGLPI/Deploy
```
Comment (cedric-anne) [inline tools/HEADER:24]:
```suggestion
@link      https://github.com/pluginsGLPI/deploy
```

File: tools/HEADER [hunk 2]
```diff
@@ -1,28 +1,25 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Deploy. If not, see <http://www.gnu.org/licenses/>.
 -------------------------------------------------------------------------
-@copyright Copyright (C) 2022 by the deploy plugin team.
-@license   MIT https://opensource.org/licenses/mit-license.php
-@link      https://github.com/pluginsGLPI/deploy
+@copyright Copyright (C) 2022-2023 by Deploy plugin team.
```
Comment (cedric-anne) [inline tools/HEADER:22]:
```suggestion
@copyright Copyright (C) 2022-2024 by Deploy plugin team.
```

File: ajax/dropdownTargets.php [hunk 3]
```diff
@@ -1,29 +1,34 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Deploy. If not, see <http://www.gnu.org/licenses/>.
+ * -------------------------------------------------------------------------
+ * @copyright Copyright (C) 2022-2023 by Deploy plugin team.
+ * @copyright Copyright (C) 2022 by the deploy plugin team.
+ * @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+ * @license   MIT https://opensource.org/licenses/mit-license.php
+ * @link      https://github.com/pluginsGLPI/Deploy
+ * @link      https://github.com/pluginsGLPI/deploy
```
Comment (cedric-anne) [inline ajax/dropdownTargets.php:30]: Please run `./vendor/bin/licence-headers-check --discard-extra-tags` to remove duplicated header tags.

**Review by cedric-anne — CHANGES_REQUESTED** [review]

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**